src/plugins/subsystems/drm/drm.sh

```diff
@@ -349,9 +372,65 @@ function get_supported_mode_per_connector()
   printf '%s\n' "$modes"
 }
 
+# Creates a binary EDID file, for each connected monitor, on the current folder.
+function get_edid()
+{
+  declare -a monitors
+
+  monitors=( $(ls -d /sys/class/drm/*/edid) )
```

Comment:
Avoid the use of `ls` since it can have different outputs in different distros. Use the `find` command. Also, check:

https://kworkflow.org/content/codingstyle.html#avoid-short-options